pkg/containerutil/containerutil.go

```diff
@@ -272,7 +272,11 @@ func Start(ctx context.Context, container containerd.Container, flagA bool, clie
 		}
 	}
 	detachC := make(chan struct{})
-	task, err := taskutil.NewTask(ctx, client, container, flagA, false, flagT, true, con, logURI, detachKeys, namespace, detachC)
+	flagAStreams := []string{}
+	if flagA {
+		flagAStreams = []string{"STDOUT", "STDERR"}
```

Comment:
Should `STDIN` also be included?

Reply:
Based on the documentation and previous implementation it should just be `STDOUT` and `STDERR`.



> -a, --attach: Attach STDOUT/STDERR and forward signals [(from nerdctl start command reference)](https://github.com/containerd/nerdctl/blob/main/docs/command-reference.md#whale-nerdctl-start)